### ISSUE TITLE
Dark Theme display issue: HTML Preview not displaying inline code markup with highlight background color 

### ISSUE THREAD

**Issue by darianmiller** [ideasawakened/UniversalMarkupEditor#10]
Multi-line code is displayed as expected but `inline` code is not properly highlighted

**Comment by darianmiller** [comment]
We use CSS from https://github.com/sindresorhus/github-markdown-css which is the basis for the "light" theme.  The dark theme is an edited version and the background color of this CSS element was missed.

before
```
.markdown-body code {
  padding: .2em .4em;
  margin: 0;
  font-size: 85%;
  background-color: rgba(27,31,35,.05);
  border-radius: 3px;
}
```
after
```
.markdown-body code {
  padding: .2em .4em;
  margin: 0;
  font-size: 85%;
  background-color: #f6f8fa;
  color: #000000;
  border-radius: 3px;
}
```

**Comment by darianmiller** [comment]
Will be fixed in version 1.1
